sms_toolkit/__init__.py

```diff
@@ -0,0 +1 @@
+from sms_toolkit.messages.normalization import normalize_optimistic_gsm_characters
```

Comment:
Can you add an import here for the profile method too, just so its easy to import?